docs/pages/machine-id/access-guides/terraform.mdx

```diff
@@ -173,6 +173,91 @@ Check your Teleport cluster, ensuring the role has been created:
 $ tctl get role/terraform-test
 ```
 
+## Example: Github Actions
+
+Let's look at this in a concrete example of running Terraform in CI using GitHub Actions.
+
+First we'll create a token and bot that allow jobs running in Actions in a specific organization,
```

Comment:
We don't tell the user how to create these resources, we just include the YAML and expect them to know what to do next.

Are we missing a tctl command?

Reply:
Yes, that's a good call out, I will make sure to include that next time I do something like this.

Removing this whole addition for now at Paul's suggestion. Resolving.